cli.js

```diff
@@ -42,6 +42,11 @@ const repo = cli.input[0];
 			process.exit(1);
 		}
 
-		await open(url);
+		if (!url) {
+			console.error('Couldn\'t find the repo\'s GITHUB URL. Ensure you are inside a github repo.');
```

Comment:
This is not what I wrote in the suggestion.

Reply:
fixed